### PR TITLE
Check GitHub Username

### LINKED ISSUE #87: Wrong Username
When I enter a wrong GitHub username instead of error it send a verification mail.

Screenshot (25)
Screenshot (26)

### PR DESCRIPTION
**Pull request by pAditya198**
Fixes #87 

Using GitHub search API I fetched if the total count of the username is 1 or not, if yes then the email verification is to be send otherwise shows an error.

Screenshot (27)

with existing username

Screenshot (28)

with wrong username.

### CODE REVIEW

File: public/client.js [hunk 1]
```diff
@@ -70,35 +85,55 @@ $(function () {
     const username = $('#username').val()
     const email = $('#email').val()
     console.log(username)
```
Comment (yashLadha) [inline public/client.js]: Is this necessary?

File: public/client.js [hunk 2]
```diff
@@ -70,35 +85,55 @@ $(function () {
...
-      .then((res) => {
-        console.log(res)
-        if (res.status === 200) {
+    fetch(`https://api.github.com/search/users?q=${username}`)
+      .then(res => res.json())
+      .then((out) => {
+        if (out.total_count === 1) {
+          fetch(`/sendmail/${username}/${email}`)
+            .then((res) => {
+              console.log(res)
```
Comment (yashLadha) [inline public/client.js]: Try to avoid `logs` when sending the final code.

File: public/client.js [hunk 3]
```diff
@@ -70,35 +85,55 @@ $(function () {
...
-        console.error(e)
-        // eslint-disable-next-line
-        UIkit.notification({
-          message: '<span class=\'uk-text-small\' uk-icon=\'icon: warning\'>An error occured. Please try again later.</span>',
-          status: 'danger',
-          pos: 'top-center',
-          timeout: 1000
-        })
-      })
+      }).catch(err => console.error(err))
```
Comment (yashLadha) [inline public/client.js]: How the user will be able to see this error? Maybe setting to an unexpected message would help though.

File: public/client.js (hunk 2 above)
Comment (pAditya198) [reply public/client.js]: will take care of this in the future

File: public/client.js (hunk 3 above)
Comment (pAditya198) [reply public/client.js]: I'll work on it.

**Comment by pAditya198** [comment]
@aashutoshrathi 

> Since we are already checking for username, shouldn't we also disable the button if any of field is invalid?

Sir here what you want is to check for validation of the email as well or if any of the fields are invalid the request is further processed

**Pull request merged**